docs/09-英文/02-Producer/07message3.md

```diff
@@ -0,0 +1,45 @@
+# Delay Message Sending
+
+The delay message sending means that when a message is sent to Apache RocketMQ, instead of delivering the message immediately, it is delivered to the Consumer for consumption after delaying a certain period of time.
```

Comment:
> The delay message sending means that when a message is sent to Apache RocketMQ, instead of delivering the message immediately, it is delivered to the Consumer for consumption after delaying a certain period of time.

Would better rephrase to:

> The delay message sending means that when a message is sent to Apache RocketMQ, instead of delivering the message immediately, it would be delivered to the Consumer for consumption after delaying a certain period of time.